{{cookiecutter.project_dirname}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -201,7 +197,7 @@ class Local(ProjectDefault):
     except ModuleNotFoundError:  # pragma: no cover
         pass
     else:  # pragma: no cover
-        INTERNAL_IPS = values.ListValue([], environ_name="ALLOWED_HOSTS")
+        INTERNAL_IPS = values.ListValue(["127.0.0.1"])
```

Comment:
What I am proposing here is to stop mapping `INTERNAL_IPS` (https://docs.djangoproject.com/en/4.0/ref/settings/#internal-ips) to `ALLOWED_HOSTS`, as it expects an IP address and not a hostname. This actually appears to be already half-implemented, since the IP address is nowhere to be found in the `DJANGO_ALLOWED_HOSTS` environment variable: https://github.com/20tab/django-continuous-delivery/blob/master/%7B%7Bcookiecutter.project_dirname%7D%7D/.env_template. All is left to do is therefore to assign `INTERNAL_IPS` an IP address to have Django Debug Toolbar work again.